libraries/AP_Scripting/examples/servo_override.lua

```diff
@@ -0,0 +1,21 @@
+-- Testing set_output_pwm_chan_timeout_ms
+--
+-- This will set MAIN1 servo to 1700 pwm for 1 second,
+-- then normal behavior for 1 second, and then 1100 pwm for 1 second.
+
+local flipflop = true
+
+function update()
+    if flipflop then
+        SRV_Channels:set_output_pwm_chan_timeout_ms(0, 1700, 1000)
+        gcs:send_text(6, "flip---")
+    else
+        SRV_Channels:set_output_pwm_chan_timeout_ms(0, 1100, 1000)
+        gcs:send_text(6, "---flop")
+    end
+    flipflop = not flipflop
+    return update, 2000
```

Comment:
doesn't this set the  duration to 2 seconds for both states?

Reply:
See other comment on line 10.